<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAddEdit</class>
 <widget class="QDialog" name="DialogAddEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="labelHead">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelMan">
       <property name="text">
        <string>Производитель</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditMan"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelModel">
       <property name="text">
        <string>Модель</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditModel"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelType">
       <property name="text">
        <string>Тип механизма</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboType">
       <item>
        <property name="text">
         <string>Мембранная</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Механическая</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Мехомембранная</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelConstr">
       <property name="text">
        <string>Тип конструкции</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="comboConstr">
       <item>
        <property name="text">
         <string>Классический</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Эргономичный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Раздельный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ромбический</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelInterface">
       <property name="text">
        <string>Тип подключения</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="comboInterface">
       <item>
        <property name="text">
         <string>PS2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>USB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Безпроводной</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="labelKolVo">
       <property name="text">
        <string>Количество клавиш</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="lineEditKolVo"/>
     </item>
     <item row="7" column="0">
      <widget class="QGroupBox" name="groupDopKl_2">
       <property name="title">
        <string>Наличие подсветки</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="radioYes_2">
          <property name="text">
           <string>Да</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioNo_2">
          <property name="text">
           <string>Нет</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QGroupBox" name="groupDopKl">
       <property name="title">
        <string>Наличие доп.клавиш</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="radioYes">
          <property name="text">
           <string>Да</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioNo">
          <property name="text">
           <string>Нет</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="labelPrice">
       <property name="text">
        <string>Цена</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="lineEditPrice"/>
     </item>
     <item row="9" column="1">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogAddEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogAddEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
